Tests/FluentAssertions.Equivalency.Specs/DataSpecs.cs

```diff
@@ -591,12 +591,16 @@ public enum ChangeType
     public static IEnumerable<object[]> AllChangeTypes =>
         Enum.GetValues(typeof(ChangeType)).Cast<ChangeType>().Select(t => new object[] { t });
 
-    public static IEnumerable<object[]> AllChangeTypesWithAcceptChangesValues =>
-        Enum.GetValues(typeof(ChangeType)).Cast<ChangeType>().Join(
-            new[] { true, false },
-            changeType => true,
-            acceptChanges => true,
-            (changeType, acceptChanges) => new object[] { changeType, acceptChanges });
+    public static IEnumerable<object[]> AllChangeTypesWithAcceptChangesValues
+    {
+        get
+        {
+            return
+                from changeType in Enum.GetValues(typeof(ChangeType)).Cast<ChangeType>()
+                from acceptChanges in new[] { true, false }
+                select new object[] { changeType, acceptChanges };
+        }
+    }
```

Comment:
A simpler way to make a Cartesian product and fixes analyzers about `changeType` and `acceptChanges` parameters in `changeType => true` and `acceptChanges => true` being unused.